git-imerge

```diff
@@ -3640,8 +3653,27 @@ def cmd_reparent(parser, options):
     except ValueError as e:
         sys.exit(e.message)
 
-    sys.stdout.write('%s\n' % (git.reparent(commit_sha1, parent_sha1s),))
```

Comment:
It is not OK to remove this output or complicate it. The `reparent` command is pretty useless if you don't do anything with its output (because it doesn't change any references itself). And likely people want to use the output directly; e.g.,

    git update-ref refs/heads/mynewbranch $(git imerge reparent $PARENT)

This works with the old code because the only thing that it outputs to stdout is the final SHA-1. But it won't work with your changes.